main/webapp/modules/core/scripts/project/process-panel.js

```diff
@@ -230,7 +230,8 @@ ProcessPanel.prototype._perform = function(jobs) {
         ui.browsingEngine.addFacet(
             job.facetType,
             job.facetConfig,
-            job.facetOptions
+            job.facetOptions,
+            true
```

Comment:
The connection of this to reconciliation is totally non-obvious to me, but I'm going to assume that this is correct.

Reply:
There are only two operations (in core and bundled extensions) which make use of the `createFacet` action to be run after the operation completes: reconciliation and Wikibase upload. This part of the code handles such an action, and by adding this `true` boolean we are making sure that facets are deduplicated before being added.